feat: adding dependencies example for insight (#4)
Decathlon/pull-request-labeler-action: README.md

  labels: ["database", "critical"]
- regExp: ".*\\.md+$"
  labels: ["documentation"]
- regExp: "^(pom\\.xml|package\\.json|build\\.gradle)$"
  labels: ["dependencies"]
- regExp: ".*\\.(zip|jar|war|ear)+$"
  labels: ["artifact", "invalid"]
```

> _this is a sample to illustrate how it works, you can customize your settings by adding your own regex_

Reviewer: ```suggestion
_This is a sample to illustrate how you can customize your settings by configuring your own regex and labels_
```
